src/emulator/instruction.ts

```diff
@@ -28,6 +29,7 @@ export const enum Operation {
     ld,
     ldd,
     ldi,
+    lds,
```

Comment:
Siehe oben, ich würde entweder `lds` + `imm8` oder `ld` + `imm8stack` machen. Beides zusammen ist irgendwie redundant.

Reply:
imm8 liest den Wert so wie er ist.  imm8stack addiert den Wert auf den Stackpointer. LDS siehe oben.

Reply:
Das kann dann aber ja direkt einfach nach `opLds` wandern, ohne einen zusätzlichen Adressierungsmodus.

Reply:
Den Addressierungsamodus nutzt ja nicht nur LDS sondern jetzt auch ADD und JR. ;)

Reply:
`imm8stack` wird nur von `LDS` benutzt. Du meinst `imm8sign`, der ist in meinen Augen aber auch nicht wirklich notwendig (siehe unten).

Reply:
Stimmt, hab ich jetzt verwechelt ;)